src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/AnonymousTypeManager.Templates.cs

```diff
@@ -91,8 +91,10 @@ public override int GetHashCode()
         private readonly ConcurrentDictionary<Location, bool> _sourceLocationsSeen = new ConcurrentDictionary<Location, bool>();
 #endif
 
+#pragma warning disable // CA1822 Mark members as static - instance members are accessed in debug.
```

Comment:
If we take this change, maybe we should resolve the warning by moving the `#if DEBUG` to surround the entire method declaration.